ui_components/src/home/Home.js

```diff
@@ -559,6 +511,36 @@ function NewsSection({ newsRef, newsPosts, homeLinks }) {
         return formattedDate;
     }
 
+    function cleanNewsContents(contents) {
+        console.log(contents);
+        const parser = new DOMParser();
+        const decodedString = parser.parseFromString(contents.replace(/[\r\n]+/g, " ").replace(/<br\s*\/?>/g, '\n'), 'text/html').body.textContent;
+        return decodedString;
+    }
+
+    function saveNewsItem(postId) {
+        console.log(contentRef.current.value);
```

Comment:
Could you pull these `console.log` calls into a `debug` function so we can have it off by default, but turn on with a console var or in dev?

Example:
https://github.com/elementum-games/shinobi-chronicles/blob/46849f675f6235bd65c2cdf0bbaf5b5c48f5a3ca/ui_components/src/travel/Travel.js#L60-L64

Reply:
Addressing in follow-up PR.